spec/features/snap_app_spec.rb

```diff
@@ -251,6 +256,11 @@
       click_on "Continue"
     end
 
+    on_page "What else should we know?" do
```

Comment:
maybe on this one we do not fill in to test at feature level that we do not need data in this field?